devsandbox-quickstarts/rhosak-devsandbox-getting-started-quickstart.yaml

```diff
@@ -5,112 +5,114 @@ metadata:
+
+        1. In the **Kafka Instances** page of the web console, for the relevant Kafka instance that you want to connect to, 
+        select the options icon (three vertical dots) and click **View connection information**.
+
+        1. In the **Connection** page, copy the **Bootstrap server** endpoint to a secure location. 
+        This is the bootstrap server endpoint that you'll need for connecting to this Kafka instance.
+
+        1. Click **Create service account** to generate the credentials that you'll use to connect to this Kafka instance.
+
+        1. Copy the generated **Client ID** and **Client Secret** to a secure location.
+
+            IMPORTANT: The generated credentials are displayed only one time, 
+            so ensure that you have successfully and securely saved the copied credentials before closing the credentials window.
+
+        1. After you save the generated credentials to a secure location, select the confirmation check box in the credentials window and close the window.
+
+            You'll use the server and client information that you saved to configure your application to connect to your Kafka instances when you're ready. 
+            For example, if you plan to use [Kafkacat](https://github.com/edenhill/kafkacat) to interact with your Kafka instance, 
+            you'll use this information to set your bootstrap server and client environment variables.
+      review:
+        instructions: |-
+          Did you save the bootstrap server endpoint to a secure location?
+
+          Did you save the client credentials to a secure location?
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+      summary:
+        success: >-
+          You have completed this task!
+        failed: Try the steps again.
+    - title: Creating a Kafka topic in Streams for Apache Kafka
+      description: >-
+        When you have a Kafka instance available, you can create Kafka topics to start producing and consuming messages in your services.
+
+        1. In the **Kafka Instances** page of the web console, click on the name of the Kafka instance that you want to add a topic to.
+
+        1. Click **Create topic** and follow the guided steps to define the topic details. 
+        Click **Next** to complete each step and click **Finish** to complete the setup.
+
+            **Topic name**: Enter a unique topic name, such as ```my-first-kafka-topic```.
+
+            **Partitions**: Set the number of partitions for this topic. This example sets the partition to ```1``` for a single partition. 
+            Partitions are distinct lists of messages within a topic and enable parts of a topic to be distributed over multiple brokers in the cluster. 
+            A topic can contain one or more partitions, enabling producer and consumer loads to be scaled.
+
+            NOTE: You can increase the number of partitions later, but you cannot decrease them.
```

Comment:
```suggestion
```
Not true anymore, is not possible to edit the partitions after creating the topic, it will be enable in the future